WorkspaceServer/Kernel/CSharpKernel.cs

```diff
@@ -323,9 +324,10 @@ bool ShouldRebuild()
             }
         }
 
-        public async Task LoadExtensionsInDirectory(IDirectoryAccessor directory, KernelInvocationContext context)
+        public async Task LoadExtensionsInDirectory(DirectoryInfo directory, KernelInvocationContext context)
```

Comment:
Since this doesn't rely directly on private state, this is a good candidate for an extension method.

Is this code C# specific?

Reply:
This method is needed for implementing the interface `IExtensibleKernel`